improve cloud section (#55)
ietf-wg-wimse/draft-ietf-wimse-workload-identity-practices: draft-ietf-wimse-workload-identity-practices.md

Within a cloud provider, the issued credential can often directly be used to
access resources of any kind across the platform, making integration between the

> services easy and "credential-less". While the term is technically misleading,
> from a user perspective, no credential needs to be issued, provisioned, rotated

Reviewer: ```suggestion
services straightforward. From the workload perspective, no credential needs to be issued, provisioned, rotated
```
